src/core/utils/heights.ts

```diff
@@ -0,0 +1,18 @@
+/*
```

Comment:
This stuff is also from `docs`, it helps to calculate responsive heights based upon set element heights. There're more constants on docs involving inkeep buttons and such.